File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
import java.util.StringTokenizer;
public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        int[] arr = new int[n];
        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;
        
        StringTokenizer st = new StringTokenizer(br.readLine());
        for(int i = 0; i < n; i++) {
            arr[i] = Integer.parseInt(st.nextToken());
        }
        for(int i : arr) {
            min = Math.min(min, i);
            max = Math.max(max, i);
        }
        
        System.out.print(min + " " + max);
    }
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.py
# n 입력받기
# n개의 정수 공백으로 구분하여 입력받기
# 최솟값, 최댓값 출력
import sys
n = int(input())
arr = list(map(int, sys.stdin.readline().split()))
print(min(arr), max(arr))